src/debug/di/rsthread.cpp

```diff
@@ -1493,7 +1493,11 @@ void CordbThread::Get32bitFPRegisters(CONTEXT * pContext)
 
     for (i = 0; i <= floatStackTop; i++)
     {
+#if !defined(FEATURE_PAL)
         long double td;
+#else
+        double td = 0.0;
+#endif
         __asm fstp td // copy out the double
```

Comment:
Does this inline assembly compile on Unix? I would expect that this whole block will need `#ifdef _MSCVER` to differentiate between Windows-style inline assembly and Unix-style inline assembly.